src/renderer/features/matrix/MatrixAutoLogin/model/matrix-autologin-model.ts

```diff
@@ -25,68 +26,90 @@ const loginFromCacheFx = createEffect((matrix: ISecureMessenger): Promise<void>
 });
 
 sample({
-  clock: loginStarted,
-  fn: () => AutoLoginStatus.IN_PROCESS,
-  target: $autoLoginStatus,
-});
-
-split({
-  source: loginStarted,
-  match: {
-    withToken: (value) => Boolean(value),
-  },
-  cases: {
-    withToken: loggedInWithToken,
-    __: loggedInFromCache,
+  clock: loggedInFromCache,
+  source: {
+    matrix: matrixModel.$matrix,
+    loginStatus: matrixModel.$loginStatus,
+    autoLoginStatus: $autoLoginStatus,
```

Comment:
Why do you need `autoLoginStatus`? as i see you don't use in filter or fn

Reply:
We use it in `ui/MatrixAutoLogin` to show animation status